pinot-servicemanager/install.sh

```diff
@@ -5,7 +5,7 @@
 set -eux
 
 # Choose the main distribution and the plugins we use
-for artifactId in pinot-distribution pinot-confluent-avro pinot-kafka-2.0
+for artifactId in pinot-distribution pinot-confluent-avro pinot-avro pinot-kafka-2.0
```

Comment:
Why do we need pinot-avro?
Also do we need pinot-confluent-avro now?

Reply:
Yup, in this image, we can remove it. Will try this.

Reply:
As discussed, we will keep this as we may want to switch back to schema-registry-based deployment.